content/learn/book/welcome/_index.md

```diff
@@ -1,9 +1,48 @@
 +++
-title = "Welcome to Bevy"
+title = "Welcome"
 weight = 1
 sort_by = "weight"
 template = "book-section.html"
 page_template = "book-section.html"
 +++
 
-TODO: Briefly discuss what Bevy is.
+Welcome to the Bevy game engine! A bevy may be a group of birds, but our engine is:
+
+- **Rust-first:** all the safety, tooling and power that come with a modern systems language
+- **Completely free:** [free-of-charge](https://github.com/sponsors/cart), [open source](https://github.com/bevyengine/bevy/blob/main/LICENSE), and hackable with an architecture designed for modularity
+- **Innovative ECS:** powered by an ergonomic, full-native entity-component-system architecture that makes writing clear, performant gameplay and engine logic a joy
+- **Cross-platform:** support Windows, MacOS, Linux, web*, Android* and iOS with a single code base
```

Comment:
What do you think of mentioning about future XR support? ("XR being under development" or similar)

Reply:
Also, a brief explanation of what XR is would help people who don't know it's actually VR/AR.

Reply:
I think this is a good plan.

Reply:
Something like:
- **Cross-platform:** support Windows, MacOS, Linux, web*, Android* and iOS with a single code base. XR** support under development.

** XR refers to extended reality, which is a broader concept for spatial computing such as augmented reality, virtual reality and mixed reality.